src/components/Modal/types.ts

```diff
@@ -87,6 +90,9 @@ type BaseModalProps = Partial<ModalProps> & {
 
     /** Should we apply padding style in modal itself. If this value is false, we will handle it in ScreenWrapper */
     shouldUseModalPaddingStyle?: boolean;
+
+    /** Extend from focus-trap options can help to set focus element when inital */
```

Comment:
I think we can simplify the documentation to something like this:

```suggestion
    /** Used to set the element that should receive the initial focus */
```